runtimes/talos/src/main/java/com/talosvfx/talos/runtime/vfx/modules/SpriteMaterialModule.java

```diff
@@ -55,8 +60,10 @@ public void processCustomValues () {
 	public void setAsset (String identifier) {
 		this.assetIdentifier = identifier;
 		final AssetProvider assetProvider = graph.getEffectDescriptor().getAssetProvider();
-		Sprite asset = assetProvider.findAsset(assetIdentifier, Sprite.class);
-		userDrawable.setDrawable(new TextureRegionDrawable(asset));
+		GameAsset<?> asset = assetProvider.findGameAsset(assetIdentifier, Sprite.class);
```

Comment:
You are using SPrite.class but obtaining a GameAsset<TExture> and then casting it